### ISSUE TITLE
Model 2

### ISSUE THREAD

**Issue by jacaseyclyde** [sjsu-high-z-classify/GalNet#32]
**Model architecture.**
This is very similar to Model 1 (#24), but uses the MaxPool placements of Dominguez-Sanchez 2018, instead of Dieleman 2015. They use same padding, rather than valid for most of their convolutions (the result being that the non-filter dimensions of each layer only change on MaxPooling layers in the CNN portion).

Layer | Type | Size | Features |  Kernel | Stride | Padding | Activation | Notes
--------|--------|-------|-------------|-----------|--------|------------|-------------|---------
In | Input | 69x69 | 3 | | | | | Input Image
1 | Conv2D | 69x69 | 32 | 6x6 | 1 | Same | ReLU | |
2 | Conv2D | 69x69 | 64 | 5x5 | 1 | Same | ReLU | |
3 | MaxPool2D | 34x34 | 64 | 2x2 | 2 | Valid | | |
4 | Conv2D | 34x34 | 128 | 2x2 | 1 | Same | ReLU | |
5 | MaxPool2D | 17x17 | 128 | 2x2 | 2 | Valid | | |
6 | Conv2D | 17x17 | 128 | 3x3 | 1 | Same |  ReLU | |
7 | Dense | 36,992 | | | | | | Flattened layer
8 | Dense | 64 | | | | | ReLU | |
Out | Dense | 3 | | | | | Softmax | Output prediction

**Hyperparameters**
Initial learning rate: 0.0001
Optimizer: Nadam
Loss: categorical_crossentropy
Loss Weights: None
Metrics: accuracy

**Describe alternative hyperparameters**
Optimizer: Nadam

**Additional context**
Use dropout=.5 on all hidden Dense layers.

**References**
<to be updated>
